docs/api/categories.md

```diff
@@ -0,0 +1,265 @@
+`description`      | `string`  | **Optional** A possible more verbose description about the category.
+`color`            | `string`  | **Optional** A color, preferably in RGB format.
+`image`            | `blob`    | **Optional** An image to show with the category.
+`icon`             | `blob`    | **Optional** An icon to show with the category.
+`categories`       | `object`  | **Optional** An array with category identifiers for nesting categories in a category.
+`speechChallenges` | `object`  | **Optional** An array of Speech Challenges identifiers categorized in the category.
+
+#### Request
+
+```http
+POST /categories/category HTTP/1.1
+Accept: application/json
+Content-Type: multipart/form-data; boundary=jhgd87g7Gy3d78
+
+--jhgd87g7Gy3d78
+Content-Disposition: form-data; name="id"
+
+category_3
+--jhgd87g7Gy3d78
+Content-Disposition: form-data; name="name"
+
+Category 3
+--jhgd87g7Gy3d78
+Content-Disposition: form-data; name="description"
+
+Category three. Winner of all categories. Yes.
+--jhgd87g7Gy3d78
+Content-Disposition: form-data; name="color"
+
+#e3e3e3
+--jhgd87g7Gy3d78
+Content-Disposition: form-data; name="image"; filename="img.png"
+Content-Type: image/png
+
+<blob>
+--jhgd87g7Gy3d78
+Content-Disposition: form-data; name="icon"; filename="icon.png"
+Content-Type: image/png
+
+<blob>
+--jhgd87g7Gy3d78
+Content-Disposition: form-data; name="speechChallenges"
+Content-Type: application/json
+
+["speech_1", "speech_2", "speech_3"]
+--jhgd87g7Gy3d78--
+```
+
+
+### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+Location: https://api.itslanguage.nl/categories/category_3
+
+{
+  "id": "category_3",
+  "created": "2017-01-12T09:50:20Z",
+  "updated": "2017-01-12T09:50:20Z",
+  "name": "Category 3",
+  "description": "Category three. Winner of all categories. Yes.",
+  "color": "#e3e3e3",
+  "imageUrl": "https://api.itslanguage.nl/download/UKbsMpBsXaJUsBbK",
+  "iconUrl": "https://api.itslanguage.nl/download/GdExSbs-ZVNnQUUe",
+  "categories": [],
+  "speechChallenges": ["speech_1", "speech_2", "speech_3"]
+}
+```
+
+## Update a category
+
+### URL
+
+```http
+PUT /categories/:category HTTP/1.1
+```
+
+* `category` - **Required** The category identifier.
+
+Update one or more properties of an existing category
+
+### Request parameters
+
+Name               | Type      | Description
+-------------------|-----------|------------
+`name`             | `string`  | **Optional** A name for the category.
+`description`      | `string`  | **Optional** A possible more verbose description about the category.
+`color`            | `string`  | **Optional** A color, preferably in RGB format.
+`image`            | `blob`    | **Optional** An image to show with the category.
+`icon`             | `blob`    | **Optional** An icon to show with the category.
+`categories`       | `object`  | **Optional** An array with category identifiers for nesting categories in a category.
```

Comment:
Same as the other comment. I think the back-end should manage this automatically based on the creation/deletion of subcategories.